desktop/src/renderer/src/components/ProcessFolderButton.tsx

```diff
@@ -3,9 +3,10 @@ import { useState } from "react";
 export const ProcessFolderButton = (): JSX.Element => {
 	const [workers] = useState(window.api.workers); // Preload scripts
 	const filePath = "D:\\test-folders\\100";
+	const transfer = "TR_0000_0000";
```

Comment:
```suggestion
  //const filePath = "/Volumes/Digital-Archives/test-folders/100";
	const transfer = "TR_0000_0000";
```
Can we add this for quick switching between testing for win and mac?